src/mpid/ch3/channels/nemesis/netmod/tcp/tcp_init.c

```diff
@@ -432,27 +489,37 @@ int MPID_nem_tcp_get_business_card (int my_rank, char **bc_val_p, int *val_max_s
     }
 
     len = sizeof(sock_id);
-    ret = getsockname (MPID_nem_tcp_g_lstn_sc.fd, (struct sockaddr *)&sock_id, &len);
+    ret = getsockname (MPID_nem_tcp_g_lstn_sc.fd, &sock_id, &len);
```

Comment:
I have removed it but I'm curious why it needs to be removed, all other functions are written with a space after their function name.

Reply:
The code formatting script was run on the full repository, but we left out CH3 since it's mostly in "maintenance mode" at this point. I think the PR hook still runs on files no matter where they are to avoid making them worse.

Reply:
Got it! I think maybe later someone needs to fix the code format so that future developers would not be confused.